### PR TITLE
TASK: Change EventStore schema

### PR DESCRIPTION
**Pull request by dfeyer**
- [x] EventStore Schema
- [x] Change Event / Metadata serialization
- [x] Basic filter support

Fixes: #7 https://github.com/neos/Neos.EventStore/issues/1

### CODE REVIEW

**Comment by dfeyer** [comment]
Currently fix most of the schema discussion from https://github.com/neos/Neos.EventStore.DatabaseStorageAdapter/issues/7

File: Classes/DatabaseEventStorage.php [hunk 1]
```diff
@@ -102,50 +103,58 @@ public function commit(string $streamName, array $data, int $commitVersion, \Clo
...
-                'event' => ':event',
+                'stream' => ':stream',
+                'stream_hash' => ':stream_hash',
+                'number' => ':number',
+                'type' => ':type',
+                'type_hash' => ':type_hash',
+                'payload' => ':payload',
                 'metadata' => ':metadata',
-                'recorded_at' => ':recorded_at'
+                'savedat' => ':savedat'
```
Comment (albe) [inline Classes/DatabaseEventStorage.php:125]: I always read `save dat!` XD

Comment (dfeyer) [reply Classes/DatabaseEventStorage.php:125]: Me too ;) was more happy with recordedat ... any way look weird with no camel case or underscore ;)

Comment (dfeyer) [reply Classes/DatabaseEventStorage.php:125]: I think we can use the really short `at`

**Comment by dfeyer** [comment]
Close this one, merged into the main repository

**Pull request closed without merging**